build-capi.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+set -e
+function subst_env_vars {
+    local ORIG_FILE="$1"
+    local T_FILE="$2"
+    cp "$ORIG_FILE" "$T_FILE"
+    declare -a subst=(
+      "CAPI_DIAGNOSTICS_ADDRESS=:8443"
+      "CAPI_INSECURE_DIAGNOSTICS=false"
+      "CAPI_USE_DEPRECATED_INFRA_MACHINE_NAMING=false"
+      "EXP_MACHINE_POOL=true"
+      "EXP_CLUSTER_RESOURCE_SET=true"
+      "CLUSTER_TOPOLOGY=false"
+      "EXP_RUNTIME_SDK=false"
+      "EXP_MACHINE_SET_PREFLIGHT_CHECKS=false"
+    )
+    for SUB in "${subst[@]}" ; do
+       local WHAT="${SUB%=*}"
+       local WITH="${SUB##*=}"
+       sed -i "s;\${$WHAT[^}]*};$WITH;g" "$T_FILE" 
+    done
+}
+function subst_placeholders {
+    local T_FILE="$1"
+    #sed -i -e 's/\({{\|}}\)/{{ "\1" }}/g' "$T_FILE"
+    sed -i -e 's;__\(.Values.[^_]\+\)__;{{ \1 }};g' "$T_FILE"
+    TMP_F=$(mktemp)
+    while true ; do
+        F_LINE=$(grep '[[:space:]]*\(- \|\)'"'"'*z.._.READ_FILE._:' "$T_FILE"|head -1)
+        [ -z "$F_LINE" ] && break
+        INDENT="${F_LINE%%z??_*}"
+        INDENT="${INDENT%%\'}"
+        INDENT="${INDENT%%\ \ -\ }"
+        R_FILE="${F_LINE#*z??_.READ_FILE._:\ }"
+        R_FILE="${R_FILE%\'}"
+        R_FILE="src/$PRJ/placeholders/add/$R_FILE"
+        sed -e "s;^;$INDENT;" "$R_FILE" > $TMP_F
+        sed -i -e "/^${F_LINE}$/r $TMP_F" -e "//d" "$T_FILE"
+    done
+    rm -f "$TMP_F"
+}
+function remove_cert_manager {
+    local T_FILE="$1"
+    if [ "${USE_OC_CERT=yes}" == "yes" ] ; then
+        $YQ -i 'select(.metadata.annotations."cert-manager.io/inject-ca-from"=="capi-system/capi-serving-cert").metadata.annotations."service.beta.openshift.io/inject-cabundle"="true"' "$T_FILE"
+        $YQ -i 'del(.metadata.annotations."cert-manager.io/inject-ca-from")' "$T_FILE"
+        $YQ -i 'del(.spec.conversion.webhook.clientConfig.caBundle)' "$T_FILE"
```

Comment:
same comment please use kustomize, unless there is something that kustomize cannot do OR giving us more effort to do.

Reply:
The issue is matching annotation=value when the value contains a '/' character.